tests/api/gef_session.py

```diff
@@ -2,79 +2,83 @@
 `gef.session` test module.
 """
 
-
 import os
+import pathlib
 import random
-import subprocess
+import re
+
+from tests.base import RemoteGefUnitTestGeneric
+
 from tests.utils import (
-    TMPDIR,
-    gdb_test_python_method,
     debug_target,
-    GefUnitTestGeneric,
     gdbserver_session,
-    gdb_run_cmd,
     qemuuser_session,
-    GDBSERVER_DEFAULT_HOST
+    GDBSERVER_DEFAULT_HOST,
 )
-import re
 
 
-class GefSessionApi(GefUnitTestGeneric):
+class GefSessionApi(RemoteGefUnitTestGeneric):
     """`gef.session` test module."""
 
-    def test_func_get_filepath(self):
-        res = gdb_test_python_method("gef.session.file", target=debug_target("default"))
-        self.assertNoException(res)
-        target = TMPDIR / "foo bar"
-        subprocess.call(["cp", debug_target("default"), target])
-        res = gdb_test_python_method("gef.session.file", target=target)
-        self.assertNoException(res)
-        subprocess.call(["rm", target])
+    def setUp(self) -> None:
+        self._target = debug_target("default")
```

Comment:
isn't this not needed?

Reply:
True, doesn't hurt either (for examples)